index.html

```diff
@@ -0,0 +1,212 @@
+                                
+                <div class="grid-item">
+                    <div class="profesional"><h4 class="prof-art">Profesional Art Printing Data</h4></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    
+                    <ul class="buttons">
+                        <li class="but-html">HTML</li>
+                        <li class="but-boot">Bootstrap</li>
+                        <li class="but-ruby">Ruby</li>
+                    </ul>
+                    
+                    <div class="orange-button1">
+                        <button class="see-project1">See project</button>
+                    </div>
+                </div>
+
+                <div class="grid-item">
+                    <div class="profesional"><h4 class="prof-art">Profesional Art Printing Datas</h4></div>
+                    <div class="daily"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p></div>
+                    
+                    <ul class="buttons">
+                        <li class="but-html">HTML</li>
+                        <li class="but-boot">Bootstrap</li>
+                        <li class="but-ruby">Ruby</li>
+                    </ul>
+                    
+                    <div class="orange-button1">
+                        <button class="see-project1">See project</button>
+                    </div>
+                </div>            
+            </div>
+        
+        
+    </section>
+    
+    <section id="skillset">
+        <div class="about">
+                <h4>About me</h4>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me..</p>
+                    <div class="orange-button2">
+                        <button class="see-project2">Get my resume</button>
+                    </div>                    
+                </div>            
+                <hr>
+
+            <ul class="languajes">
+                <li class="h3">Languaje</li>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+
+                <li class="h3">Framework</li>
+                <li>React</li>
+                <li>Ruby on Rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+            
+                <li class="h3">Skills</li>
+                <li>Database Management</li>
+                <li>Version Control</li>
+                <li>CLI</li>
+                <li>API design</li>
+                <li>Web Development</li>
+            </ul>
+           </section> 
+    </main>
+    <footer>
+    </footer>
```

Comment:
- [ ] [Optional] 👍 Great job on your project so far! I noticed that you don't have a footer section in your HTML structure. Since you don't have any content or specific purpose for a footer at the moment, it may be unnecessary to include an empty `<footer>` tag in your code.

📝 Keeping your code clean and concise is important for maintainability and readability. By removing unnecessary elements, such as an empty footer, you can streamline your code and improve its clarity.

🚀 Please review your project and decide whether you need to include a footer with relevant content. If not, you can safely remove the empty `<footer>` tag from your code.